palm/plugins/dbt/dbt_containerizer.py

```diff
@@ -1,10 +1,12 @@
+import os, \
```

Comment:
Per PEP8 style, this should be one line - please use black to lint so we have consistent formatting